lib/ably/rest/client.rb

```diff
@@ -122,6 +124,8 @@ class Client
       #
       # @option options [Boolean]                 :fallback_hosts_use_default  (false) When true, forces the user of fallback hosts even if a non-default production endpoint is being used
       # @option options [Array<String>]           :fallback_hosts              When an array of fallback hosts are provided, these fallback hosts are always used if a request fails to the primary endpoint. If an empty array is provided, the fallback host functionality is disabled
+      # @option options [Integer]                 :fallback_retry_timeout     (10 minutes) amount of time a REST client will continue to use a working fallback host when the primary fallback host has previously failed
```

Comment:
should say explicitly that this is in seconds, and default should be `(600 seconds)`

Reply:
Well I don't think it necessary to say seconds as it's idiomatic to use seconds, but I will add, thanks.